src/libraries/System.Net.Mail/src/System/Net/Mime/HeaderCollection.cs

```diff
@@ -22,9 +22,9 @@ internal HeaderCollection() : base(StringComparer.OrdinalIgnoreCase)
         {
         }
 
-#pragma warning disable CS8610 // Nullability of reference types in type of parameter doesn't match overridden member.
+#pragma warning disable CS8765 // Nullability of parameter 'name' doesn't match overridden member
         public override void Remove(string name)
-#pragma warning restore CS8610
+#pragma warning restore CS8765
```

Comment:
(FWIW, in situations like this I'm fine with the pragmas; the compiler is correct and we're explicitly violating the contract.  It's the false positives that bother me.)